parlai/utils/strings.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+
+# Copyright (c) Facebook, Inc. and its affiliates.
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""
+Utility functions and classes for handling text strings.
+"""
+
+
+def standardize_string(text: str) -> str:
```

Comment:
nit, let's find a more descriptive name. It makes larger assumptions than "is a string." `normalize_reply`? `normalize_english?` The assumptions are that this text is pretty much formatted by `re_tokenize`.

Reply:
Yeah, definitely agree - changed to `normalize_reply`